src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,179 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static RegistrationService registrationService;
+
+    @BeforeAll
+    static void initRegistrationService() {
+        registrationService = new RegistrationServiceImpl();
+    }
+
+    @BeforeEach
+    void addUsers() {
+        User user1 = new User();
+        user1.setLogin("user1");
+        user1.setAge(25);
+        user1.setPassword("qwertyui");
+        User user2 = new User();
+        user2.setLogin("user2");
+        user2.setAge(66);
+        user2.setPassword("asdfghj");
+        User user3 = new User();
+        user3.setLogin("user3");
+        user3.setAge(35);
+        user3.setPassword("zxcvbnm");
+        registrationService.register(user1);
+        registrationService.register(user2);
+        registrationService.register(user3);
+    }
+
+    @AfterEach
+    void clearStorage() {
+        Storage.people.clear();
+    }
+
+    @Test
+    void register_nullLogin_notOk() {
+        User user = new User();
+        user.setAge(65);
+        user.setPassword("123989789234");
+        assertThrows(RuntimeException.class, () -> {
+            registrationService.register(user);
+        });
```

Comment:
Мне кажется, лучше добавлять информативные сообещния к ошибкам, так легче будет разобраться что пошло не так в случае если тест не пройдет (давай исправим для всех тестов)
```suggestion
        assertThrows(RuntimeException.class, () -> {
            registrationService.register(user);
        }, "Expect an RuntimeException when trying to register a user with null login");
```